Source/DafnyRuntime/DafnyRuntimeJava/gradlew

```diff
@@ -242,4 +249,4 @@ eval "set -- $(
         tr '\n' ' '
     )" '"$@"'
 
-for i in 1 2 3 4 5; do "$JAVACMD" "$@" && break || sleep 15; done
```

Comment:
Note we added retrying here for all gradle commands, and the upgrade is dropping it (both here and in gradlew.bat).

I suspect this retrying has been hiding a race condition in the C# build on windows: it transitively does a `./gradlew build` on `DafnyRuntimeJava`, and I also see evidence in the logs that something is causing two concurrent builds. The error we're getting looks to be caused by a race to delete the `build/test-results/test` directory, which the retrying would silently recover from.

To at least diagnose, I'd suggest adding back the retrying in both places to see if the CI is fixed. But I'd also strongly prefer we add this retrying at a higher level (probably in other workflow *.yml files to retry issues with remote downloads) rather than hacking gradle scripts, or else this will probably bite us again in the future.